src/main/java/uk/nhs/digital/nhsconnect/nhais/outbound/OutboundQueueService.java

```diff
@@ -62,10 +62,8 @@ public void receive(Message message) throws IOException, JMSException {
             LOGGER.debug("Parsed message into object: {}", outboundMeshMessage);
             meshClient.authenticate();
             meshClient.sendEdifactMessage(outboundMeshMessage);
-        } catch (Exception e) {
-            LOGGER.error("Error while processing mesh inbound queue message", e);
```

Comment:
Can you double check how it will behave in case of an error? I think I did that on purpose as otherwise some information would be lost. Can you check that?

Reply:
Sorted out